examples/ui/scroll.rs

```diff
@@ -0,0 +1,388 @@
+                                                background_color: Color::srgb(0.05, 0.05, 0.05)
+                                                    .into(),
+                                                ..default()
+                                            })
+                                            .insert(Pickable {
+                                                should_block_lower: false,
+                                                ..default()
+                                            })
+                                            .with_children(|parent| {
+                                                for i in 0..25 {
+                                                    parent
+                                                        .spawn((
+                                                            TextBundle::from_section(
+                                                                format!("Item {}", (oi * 25) + i),
+                                                                TextStyle {
+                                                                    font: asset_server.load(
+                                                                        "fonts/FiraSans-Bold.ttf",
+                                                                    ),
+                                                                    ..default()
+                                                                },
+                                                            ),
+                                                            Label,
+                                                            AccessibilityNode(NodeBuilder::new(
+                                                                Role::ListItem,
+                                                            )),
+                                                        ))
+                                                        .insert(Pickable {
+                                                            should_block_lower: false,
+                                                            ..default()
+                                                        });
+                                                }
+                                            });
+                                    }
+                                });
+                        });
+                });
+        });
+}
+
+/// Updates the scroll position of scrollable nodes in response to mouse input
+pub fn update_scroll_position(
+    mut mouse_wheel_events: EventReader<MouseWheel>,
+    hover_map: Res<HoverMap>,
+    mut scrolled_node_query: Query<(&mut ScrollPosition, &Style)>,
+    keyboard_input: Res<ButtonInput<KeyCode>>,
+) {
+    for mouse_wheel_event in mouse_wheel_events.read() {
+        let (mut dx, mut dy) = match mouse_wheel_event.unit {
+            MouseScrollUnit::Line => (mouse_wheel_event.x * 20., mouse_wheel_event.y * 20.),
```

Comment:
Shouldn't this 20 scale factor be pulled out into a constant and used for the font size?

Reply:
Well, we won't get perfectly 20 pixel tall text out of that anyway. But what I DID do was define it as `LINE_HEIGHT` and force the scrolled children in the vertical scroll example to be exactly that height. 

![scroll-lines](https://github.com/user-attachments/assets/991efaa6-4e00-43ba-996a-f4d5f6c9ea76)

Great idea!!